docs/DeveloperGuide.adoc

```diff
@@ -244,6 +243,33 @@ _{Explain here how the data encryption feature will be implemented}_
 
 // end::dataencryption[]
 
+// tag::reminder[]
+=== Reminder Feature
+==== Implementation
+A bookmark can have a reminder, and a reminder can be used to open the bookmark.
```

Comment:
```suggestion
A bookmark can have a reminder, and a reminder can be used to open the bookmark at a specific time.
```

Reply:
I think open at a specific time is a "good to have" feature, currently it will just show on the dashboard at the order of time.